sc_cost_meter/app.py

```diff
@@ -1,42 +1,30 @@
-import json
-
-# import requests
+import logging
+import sc_cost_meter.utils as utils
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def meter_instances():
+  '''
+  Send EC2 cost info to the AWS Marketplace
+  '''
+  instances = utils.get_instances(['running'])
+  for instance in instances:
+    instance_id = instance['InstanceId']
+    tags = utils.get_tags(instance_id)
+    for tag in tags:
+      if tag['Key'] == 'marketplace:customerId':
+        customer_id = tag['Value']
+        product_code = utils.get_marketplace_product_code(tags)
+        if product_code:
+          price = utils.get_ec2_on_demand_pricing(instance)
```

Comment:
this is not "price" but "hourly_price"

Reply:
switched up to get the actual cost of the instances instead of price.